journal/original/20-chapter-17.adoc

```diff
@@ -1255,3 +1296,4 @@ G+++.+++ F.
 
 [.signed-section-context-close]
 This Prayer was written at night, the 17th of the 2nd Month, 1671.
+//THis above shouldn't be embedded, shoul it? Since it's a prayer? but it's signed!
```

Comment:
whoa! interesting....   so, i think we should add another type.  let's _do_ embed this one, and make it a type `prayer` -- I could see it being useful in other books.  i'll add it to the styleguide.  good catch! 💯